lms/templates/ccx/enrollment.html

```diff
@@ -4,15 +4,16 @@
   <form method="POST" action="ccx_invite">
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
   <h2> ${_("Batch Enrollment")} </h2>
-  <p>
-    <label for="student-ids">
+  <label for="student-ids" class="sr">${_("Email Addresses/Usernames")}</label>
+  <p id="label_student_ids">
```

Comment:
@carsongee this label is only for screen readers. For user there is placeholder in text area, but screen readers can read it. Only for screen readers i added this label with **class=sr** it will not show on screen for regular user rather when screen reader will focus on text area he will get this label as replacement of placeholder. 

The <p> is paragraph hint for users that add students in text area.